mavros/src/lib/uas_data.cpp

```diff
@@ -51,9 +51,14 @@ UAS::UAS() :
 		ros::shutdown();
 	}
 
-	// send static transform from "local_origin" (ENU) to "local_origin_ned" (NED)
-	publish_static_transform("local_origin", "local_origin_ned", Eigen::Affine3d(ftf::quaternion_from_rpy(M_PI, 0, M_PI_2)));
-	publish_static_transform("fcu", "fcu_frd", Eigen::Affine3d(ftf::quaternion_from_rpy(M_PI, 0, 0)));
+	// send static transform that are part of the tf tree as a vector
```

Comment:
```suggestion
	// Publish helper TFs used for frame transformation in the odometry plugin
```

Reply:
Not resolved.

Reply:
Are you sure? It is in the newest version for me. Also in this PR.